index.html

```diff
@@ -1,27 +1,41 @@
 <!DOCTYPE html>
-<html lang="es">
+<html class="page-setup" lang="es">
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Mi Página Web</title>
-    <link rel="stylesheet" href="estilos.css">
+    <link rel="stylesheet" href="sanitize.css" >
+    <link rel="stylesheet" href="style.css" >
 </head>
-<body>
-    <header>
-        <h1>My Notes Application</h1>
+<body class="page-setup">
+    <header class="header">
+        <h1 class="header-title">My Notes Application</h1>
     </header>
-    <main>
-        <section>
-            <form action="/submit" method="post">
-                <label for="name">Title:</label>
-                <input type="text" id="name" name="name" required>
-                <br>
-                <label for="email">Content:</label>
-                <input type="email" id="email" name="email" required>
-                <br>
-                <button type="submit">Add Note</button>
-            </form>
+
+    <main class="main-content">
+        <section class="note-section">
+            <div class="note-create">
+                <h2 class="section-title">Create a New Note</h2>
+                <form class="note-form">
+                    <label class="label">Title:</label>
```

Comment:
Add **for** attribute to _label_ tag. It will improve a11y (accessibility) to _input_ element with same id.